polylogue/importers/claude_ai.py

```
@@ -89,15 +111,13 @@ def list_claude_conversations(export_path: Path) -> List[Dict[str, Optional[str]
-            raise ValueError(
-                "Unexpected Claude export format: ZIP archive must contain a valid conversations.json file. "
-                "Make sure you're using an official Claude.ai export."
-            )
         info: List[Dict[str, Optional[str]]] = []
-        for conv in conversations:
+        for conv in _iter_conversations(convo_path):
+            if not isinstance(conv, dict):
+                raise ValueError(
+                    "Unexpected Claude export format: ZIP archive must contain a valid conversations.json file. "
+                    "Make sure you're using an official Claude.ai export."
+                )
```

Ruff (0.15.9)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)